decidim-core/lib/decidim/content_security_policy.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This class is responsible of generating the Content-Security-Policy header
```

Comment:
(I added here so it's resolvable once we actually fix it)

We should come up with an strategy regarding the default Rails CSP configuration, so we can signal/explain to newcomers that the accessor/system panel is the preferred way of tuning this. I'm referring to the `development_app/config/initializers/content_security_policy.rb`. 

This is what I have in mind (feel free to improve of course):

```ruby
# For tuning the Content Security Policy, check the Decidim documentation site
# https://docs.decidim.org/develop/en/customize/content_security_policy
```